### PR TITLE
Move - Migrating ASP.NET MVC Applications to Windows Containers

### LINKED ISSUE #2679: Migrate article from https://docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc
I think it make more sense for [this article](https://github.com/dotnet/docs/blob/master/docs/framework/docker/aspnetmvc.md
) to live in this docset as opposed to the .NET docset.

### PR DESCRIPTION
**Pull request by spboyer**
Fixes #2679

### CODE REVIEW

**Comment by spboyer** [comment]
@danroth27 How do we get a new box added for this on - https://docs.microsoft.com/en-us/aspnet/index#pivot=aspnet&panel=aspnet_tutorials ?

**Comment by Rick-Anderson** [comment]
@tdykstra  please review

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+title: Migrating ASP.NET MVC Applications to Windows Containers
+description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
+keywords: Windows Containers, Docker, ASP.NET MVC
+author: BillWagner
+ms.author: wiwagn
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: Should author in metadata change to Shayne?

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+ms.technology: dotnet-mvc
+ms.devlang: dotnet
+ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+uid: mvc/overview/deployment/docker
+---
+# Migrating ASP.NET MVC Applications to Windows Containers
+
+Running an existing .NET Framework-based application in a Windows container doesn't require any changes to your app. To run your app in a Windows container you create a Docker image containing your app and
+start the container. This topic explains
+how to take an existing [ASP.NET MVC application](http://www.asp.net/mvc)
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: This url goes to the old asp.net site

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+You start with an existing ASP.NET MVC app, then build the published assets using Visual Studio. You use Docker
+to create the image that contains and runs your app. You'll browse to the site running in a Windows container and verify the app is
+working.
+
+This article assumes a basic understanding of Docker. You can learn about Docker by reading the [Docker Overview](https://docs.docker.com/engine/understanding-docker/).
+
+The app you'll run in a container is a simple website that
+answers questions randomly. This app is a basic MVC application
+with no authentication or database storage; it lets you focus
+on moving the web tier to a container. Future topics will show how to
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: Is this promise of future articles building on this one still valid?

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+1. [Creating a publish task to build the assets for an image.](#publish-script)
+1. [Building a Docker image that will run your application.](#build-the-image)
+1. [Starting a Docker container that runs your image.](#start-a-container)
+1. [Verifying the application using your browser.](#verify-in-the-browser)
+
+The [finished application](https://github.com/dotnet/docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator) is on GitHub.
+
+## Prerequisites
+
+The development machine must be running
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: add colon

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+1. Right click on the web project in Visual Studio, and select **Publish**.
+1. Click the **Custom profile button**, and then select **File System** as the method.
+1. Choose the directory. By convention, the downloaded sample uses `bin/PublishOutput`.
+
+![Publish Connection][publish-connection]
+
+Open the **File Publish Options** section of the **Settings** tab. Select
+**Precompile during publishing**. This optimization means that you'll be
+compiling views in the Docker container, you are copying the precompiled
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: Please double-check this sentence, doesn't sound right.

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+## Build the image
+
+Define your Docker image in a Dockerfile. The Dockerfile contains instructions
+for the base image, additional components, the app you
+want to run, and other configuration images.  The Dockerfile is the input
+to the `docker build` command, which creates the image.
+
+You will build an image based on the `microsft/aspnet`
+image located on [Docker Hub](https://hub.docker.com/r/microsoft/aspnet/).
+The base image, `microsoft/aspnet`, is a Windows Server image. In contains
```
Comment (tdykstra) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: In contains --> It contains

File: aspnet/mvc/overview/deployment/docker-aspnetmvc.md (hunk 5 above)
Comment (Rick-Anderson) [inline aspnet/mvc/overview/deployment/docker-aspnetmvc.md]: @BillWagner can you comment on this?

**Pull request merged**